### PR TITLE
ADD: unity ci

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: Whist-Desktop/Librarys
+          key: Library-Whist-Desktop-${{ matrix.os }}
+          restore-keys: |
+            Library-MyProjectName-
```
Comment (iTitus) [inline .github/workflows/test.yml]:
```suggestion
          path: Whist-Desktop/Library
          key: Library-Whist-Desktop-${{ runner.os }}
          restore-keys: |
            Library-Whist-Desktop-
```

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (Segelzwerg) [reply .github/workflows/test.yml]: Dang copy paste, but why use runner here?

Comment (iTitus) [reply .github/workflows/test.yml]: its safer, maybe we want to remove the matrix, then this still works

**Review by iTitus — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (Segelzwerg) [reply .github/workflows/test.yml]: Ah didn't know this will work too

**Pull request merged**